src/components/AuthComponent/Auth.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react'
+import SignIn from '../../containers/SignInContainer'
+import SignUp from '../../containers/SignUpContainer'
+import ForgotPassword from '../../containers/ForgotPasswordContainer'
+
+export interface IAuthProps {
+  config: { [key: string]: string }
+  className?: string
+}
+
+const AuthComponent = ({
+  config,
+  className = 'a6y-react-auth',
+}: IAuthProps): JSX.Element => {
+  useEffect(() => {
+    if (config && config.currentForm) {
```

Comment:
As we spoke, we don't read it from the config. We can use `useEffect` to find a specific query param, ie: `f=sign-in`. And then if any is set, we can set it in the `currentForm`.

However, I'm wondering if it could be done inside of `useState` (I mean to read the query param). It's because that if we use it useEffect, the page (maybe) will be rendered twice. First time, before useEffect will run, and then after the useEffect sets the `currentForm`.

Reply:
For now, you can just remove `useEffect`. We can add support for query params later.